.travis.yml

```diff
@@ -26,10 +26,21 @@ install:
 - mkdir -p "${DOCKER_CONFIG}"
 - echo '{}' >"${DOCKER_CONFIG}/config.json"
 - docker login --username "${DOCKERHUB_USERNAME}" --password-stdin <<<"${DOCKERHUB_PASSWORD}"
+stages:
+  - check
+  - test
+  - build
+  - deploy
 jobs:
   include:
+  - stage: check
+    before_script:
+    - go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.33.0
```

Comment:
You might need to `( cd; go get ... )` because otherwise it'll just try to add the dependency to the current go.mod instead (I think).

Reply:
Seems to work fine (for both `golangci-lint` and `gotestsum`) based on quick testing.
Might refine this into scripting later, but I believe this should be fine for now.